libs/clients/client.go

```diff
@@ -167,8 +167,7 @@ func (c *SimpleHTTPClient) newRequest(
 		RawQuery: qs,
 	})
 
-	// m, _ := json.MarshalIndent(body, "", "  ")
-	if body != nil && method != "GET" {
+	if body != nil {
```

Comment:
I think if we fix the cohort passing via body on GET as mentioned above we can undo this